source/tools/1passwordEntry.js

```diff
@@ -44,6 +44,13 @@ function onePasswordItemToEntry(rawItem) {
     return entry;
 }
 
+function testNormalEntryAndFieldsExist(rawItem) {
+    return (
+        ENTRY_NORMAL.test(rawItem.typeName) &&
+        typeof rawItem.secureContents.fields !== "undefined"
```

Comment:
since we are doing `forEach` on this property, isn't it better to check if it's an array or not? eg `Array.isArray(rawItem.secureContents.fields)`

Reply:
I agree and changed the test